### PR TITLE
Added props to storybook

### PR DESCRIPTION
**Pull request by rijusougata13**
## Description

Added props to storybook
## How Has This Been Tested?

with npm
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/Dashboard/index.js [hunk 1]
```diff
@@ -49,20 +60,31 @@ const Dashboard = () => {
   const [nameValidateErrorMessage, setNameValidateErrorMessage] = useState("");
   const [orgName, setOrgName] = useState("");
   const [orgNameValidateError, setOrgNameValidateError] = useState(false);
-  const [orgNameValidateErrorMessage, setOrgNameValidateErrorMessage] = useState("");
+  const [
+    orgNameValidateErrorMessage,
+    setOrgNameValidateErrorMessage,
+  ] = useState("");
```
Comment (shehand) [inline src/components/Dashboard/index.js:66]: could be in a single line

File: src/components/Dashboard/index.js [hunk 2]
```diff
@@ -49,20 +60,31 @@ const Dashboard = () => {
...
   const [handle, setHandle] = useState("");
   const [handleValidateError, setHandleValidateError] = useState(false);
-  const [handleValidateErrorMessage, setHandleValidateErrorMessage] = useState("");
+  const [handleValidateErrorMessage, setHandleValidateErrorMessage] = useState(
+    ""
+  );
```
Comment (shehand) [inline src/components/Dashboard/index.js:71]: could be in a single line

File: src/components/Dashboard/index.js [hunk 3]
```diff
@@ -49,20 +60,31 @@ const Dashboard = () => {
...
   const [orgHandle, setOrgHandle] = useState("");
   const [orgHandleValidateError, setOrgHandleValidateError] = useState(false);
-  const [orgHandleValidateErrorMessage, setOrgHandleValidateErrorMessage] = useState("");
+  const [
+    orgHandleValidateErrorMessage,
+    setOrgHandleValidateErrorMessage,
+  ] = useState("");
```
Comment (shehand) [inline src/components/Dashboard/index.js:77]: could be in a single line

File: src/components/Dashboard/index.js [hunk 4]
```diff
@@ -49,20 +60,31 @@ const Dashboard = () => {
...
   const [countryValidateError, setCountryValidateError] = useState(false);
   const [orgCountry, setOrgCountry] = useState("");
   const [orgCountryValidateError, setOrgCountryValidateError] = useState(false);
   const [orgWebsite, setOrgWebsite] = useState("");
   const [orgWebsiteValidateError, setOrgWebsiteValidateError] = useState(false);
-  const [orgWebsiteValidateErrorMessage, setOrgWebsiteValidateErrorMessage] = useState("");
+  const [
+    orgWebsiteValidateErrorMessage,
+    setOrgWebsiteValidateErrorMessage,
+  ] = useState("");
```
Comment (shehand) [inline src/components/Dashboard/index.js:87]: could be in a single line

File: src/components/Dashboard/index.js [hunk 5]
```diff
@@ -115,9 +137,22 @@ const Dashboard = () => {
...
-      if (nameValid && orgNameValid && countryValid && orgCountryValid && orgWebsiteValid) {
+      const orgCountryValid = validateCountry(
+        orgCountry,
+        setOrgCountryValidateError
+      );
+      const orgWebsiteValid = validateOrgWebsite(
+        orgWebsite,
+        setOrgWebsiteValidateError,
+        setOrgWebsiteValidateErrorMessage
+      );
```
Comment (shehand) [inline src/components/Dashboard/index.js:148]: could be in a single line

File: src/components/Dashboard/index.js [hunk 6]
```diff
@@ -115,9 +137,22 @@ const Dashboard = () => {
...
+      if (
+        nameValid &&
+        orgNameValid &&
+        countryValid &&
+        orgCountryValid &&
+        orgWebsiteValid
+      ) {
```
Comment (shehand) [inline src/components/Dashboard/index.js:155]: could be in a single line

File: src/stories/AuthPage/Signup/index.stories.js [hunk 7]
```diff
@@ -1,16 +1,27 @@
...
+      <SignUp {...args} />{" "}
     </MemoryRouter>
   </ProviderWrapper>
 );
+
+export const Default = Template.bind({});
+
+// Default.args = {
+//   background: "white",
+// };
```
Comment (shehand) [inline src/stories/AuthPage/Signup/index.stories.js:27]: if not needed, remove this.

**Review by shehand — CHANGES_REQUESTED** [review]

File: src/components/Dashboard/index.js (hunk 1 above)
Comment (rijusougata13) [reply src/components/Dashboard/index.js:66]: actually vscode prettier formating this lines . I'll edit this in different editor and commit those.

**Review by shehand — APPROVED** [review]

**Review by Niweera — APPROVED** [review]

**Comment by rijusougata13** [comment]
Fix #157

**Pull request merged**